portal/apps/org.wso2.carbon.iam.admin-portal.app/src/main/public/js/add.js

```diff
@@ -10,13 +10,21 @@ $(document).ready(function () {
     });
     $('#verificationSelector').change(displayVals);
 });
+
 $(window).load(function () {
     jQuery.validator.addMethod("noSpace", function (value, element) {
         return value.indexOf(" ") < 0 ;
     }, "Please don't enter spaces");
 });
+
+window.setTimeout(function () {
+    $(".alert-success").fadeTo(500, 0).slideUp(500, function () {
+        $(this).remove();
+    });
+}, 5000);
+
 function displayVals() {
-    if ($("#verificationSelector option:selected").text() === "With password") {
+    if ($("#verificationSelector").children(":selected").attr("id") == "with_password") {
```

Comment:
@denuwanthi Looks like we can rewrite this with "switch" and most probably reuse the code
"var fillingObject = {};
         var callbacks = {
             onSuccess: function () {
             },
             onFailure: function (e) {
             }
         };
         UUFClient.renderFragment("ask-password", fillingObject,
             "accountVerificationMethod-area", "OVERWRITE", callbacks)